Sources/Experiment/ExperimentClient.swift

```diff
@@ -15,6 +15,7 @@ import Foundation
     @objc func variant(_ key: String, fallback: Variant?) -> Variant
     @objc func all() -> [String:Variant]
     @objc func exposure(key: String)
+    @objc func exposure(key: String, variant: String, payload: Any?)
```

Comment:
`@objc func exposure(key: String, variant: Variant)`
To better match the rest of the API. Tt looks strange to pass a String instead of an actual Variant.